### ISSUE TITLE
userConfiguration byte 3 bit 6 is Sabbath Mode

### ISSUE THREAD

**Issue by Hixie** [GEMakers/gea-plugin-dishwasher#8]
The documentation says, under `User configuration`, that bits "6+" for the third byte are "reserved". As far as I can tell, at least on the GE GDF570SGFWW dishwasher, bit 6 (0x40) is Sabbath Mode (hold start and wash temp buttons for five seconds with door closed to toggle).

**Comment by Hixie** [comment]
The manual says it's supposed to turn off automatically when you press a button, but the bit doesn't toggle then. You have to manually toggle it. It just seems to control whether or not the dishwasher wakes up when you open it.

**Comment by johnnysako** [comment]
@Hixie You are correct. "Sabbath" mode enables or disables the UI waking when the door is opened or closed. If enabled a button press is required to wake the UI.
